packages/flutter_tools/gradle/src/main/groovy/flutter.groovy

```diff
@@ -765,6 +765,7 @@ class FlutterPlugin implements Plugin<Project> {
         if (pluginProject == null) {
             return
         }
+        pluginProject.extensions.create("flutter", FlutterExtension)
```

Comment:
Consider adding a comment saying what this line does for future maintainers.

Reply:
done!